Consolidate business/balances cards in Business and Deposit&Withdrawal routes

jurgenwerk opened the pull request:
Ticket: [CS-2890](https://linear.app/cardstack/issue/CS-2890/leverage-business-account-form-from-depositwithdrawal-page-on)

This PR consolidates the "business&balances" components from both routes, and uses the same component to display depot and merchant balances info. 

Business route:

![image](https://user-images.githubusercontent.com/273660/148751292-9db3f532-c55f-4b5a-900c-350df6381a19.png)

Deposit & Withdrawal: 

![image](https://user-images.githubusercontent.com/273660/148751337-e146ee5b-d000-4fbb-8ddf-06f06a69c502.png)

jurgenwerk commented on packages/web-client/app/templates/card-pay/payments.hbs:
```diff
@@ -69,11 +69,7 @@
-            @waitForCustomization={{true}}
-            data-test-merchant-dashboard-card
-          />
+          <CardPay::SafeBalances @safe={{merchantSafe}} />
```
This component is now used in both /payments and /deposit-withdrawal routes. 

Now that this component contains the safe info, together with balances, would it make sense to rename it to `CardPay::Safe` (and all the CSS that goes with it)?

lukemelia replied:
I like that idea!

jurgenwerk commented:
What I'm not sure about is if we should be displaying the safe footer in the Depot ... Will ask Kyle

![image](https://user-images.githubusercontent.com/273660/148758631-7b020507-da58-46be-ac2f-8e90e1de0bd2.png)

Aierie reviewed (changes requested):
I think this works! Besides the question you've raised earlier, I think it might be good to clarify if the "footer" is meant to be placed below the business' logo with this new change, since that is the order that it appears in the spec and it seems to make sense to put a business' identifiers together. 

Requested change: `CardPay::MerchantSafe` is no longer used + references the removed component `CardPay::MerchantCard` and should be removed.

Aierie commented:
> What I'm not sure about is if we should be displaying the safe footer in the Depot ... I asked Kyle
> 
> ![image](https://user-images.githubusercontent.com/273660/148758631-7b020507-da58-46be-ac2f-8e90e1de0bd2.png)

Is it useful to the user to show "1 Manager" for either of these? It feels like it isn't.

lukemelia commented:
> Is it useful to the user to show "1 Manager" for either of these? It feels like it isn't.

Not useful in either case right now, but in the case of the Depot it doesn't make any sense period, and in the case of the Business, it may later when we support multiple managers.

jurgenwerk commented:
@Aierie thanks for pointing out the dead code, I removed it. Regarding whether the footer should really be a footer vs. being in the middle (as in the original spec) - I thought that placing it on the bottom looks better, and Kyle commented in the ticket it looks ok. So I think we can go with this.

@lukemelia thanks for the tip, I removed that line from the depot safe but kept it in others. Let me know if that isn't ok.

Aierie reviewed (approved):
Looks good! Thanks for checking w design 😄
